zone/inventory.cpp

```
@@ -1079,10 +1117,14 @@ bool Client::PutItemInInventory(int16 slot_id, const EQ::ItemInstance& inst, boo
 
 	if (slot_id == EQ::invslot::slotCursor) {
 		auto s = m_inv.cursor_cbegin(), e = m_inv.cursor_cend();
```

Clang (14.0.6)

[warning] multiple declarations in a single statement reduces readability

(readability-isolate-declaration)